corgi/collectors/brew.py

```diff
@@ -684,7 +684,10 @@ def get_component_data(self, build_id: int) -> dict:
             # If this is an "image" type build, it may be building a container image, ISO image,
             # or other types of images. Check the content generator name to determine where this
             # image was built, which indicates what type of image it is.
-            if build["cg_name"] != "atomic-reactor":
+            # Some builds such as 903565 have the cg_name field set to None
+            if build["cg_name"] is None or build["cg_name"] == "atomic-reactor":
```

Comment:
@jasinner I'm not sure this is actually a good idea. Some old stuff might have not set the content generator value but will probably not be a container image build. Maybe we should refactor this to check if the build has a `buildContainer` task? That would be a good indicator whether its a container image build or some random ISO image build. Wdyt?

Reply:
I change the check from `build["cg_name"] == None` to `build["extra"]["submitter"] == "osbs"` instead.